package.json

```diff
@@ -18,6 +18,7 @@
     "@rollup/plugin-commonjs": "14.0.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-node-resolve": "8.4.0",
+    "rollup-plugin-terser": "7.0.2",
```

Comment:
Failed to discover the version of terser used by bazel, so picked the latest version of this plugin. Will take a further look trying to align as much as possible